backend/tracim_backend/tests/library/test_user_api.py

```diff
@@ -76,23 +65,24 @@ def test__unit__create__user__ok_nominal_case(self, session, app_config):
     "config_section", [{"name": "functional_test_with_mail_test_sync"}], indirect=True
 )
 class TestUserApiWithNotifications:
-    @pytest.mark.parametrize("with_email", (True, False))
+    @pytest.mark.parametrize("email", ("bob@bob.local", None))
     def test__unit__create_user__ok__with_or_without_email(
-        self, session, app_config, with_email: bool,
+        self, session, app_config, email: typing.Optional[str],
     ):
+        app_config.EMAIL__REQUIRED = False
```

Comment:
same question, is there a reason here to set config in test ?

Reply:
Corrected.